defaults/main.yml

```diff
@@ -2,7 +2,6 @@
 # defaults file for ansible-role-oiofs
 
 openio_oiofs_packages:
-  - openio-sds-common
```

Comment:
Are you sure we don't need this package anymore? The problem was originally that the package`openio-sds-common` doesn't exist on Ubuntu.